lib/internal/test_runner/utils.js

```diff
@@ -250,45 +263,130 @@ function countCompletedTest(test, harness = test.root.harness) {
 }
 
 
-function coverageThreshold(coverage, color) {
-  coverage = NumberPrototypeToFixed(coverage, 2);
-  if (color) {
-    if (coverage > 90) return `${green}${coverage}${color}`;
-    if (coverage < 50) return `${red}${coverage}${color}`;
-  }
-  return coverage;
+function addTableLine(prefix, width) {
+  return `${prefix}${'-'.repeat(width)}\n`;
+}
+
+function truncateStart(string, width) {
+  return string.length > width ? `\u2026${string.substring(string.length - width + 1, string.length)}` : string;
```

Comment:
```suggestion
  return string.length > width ? `\u2026${StringPrototypeSubstring(string.length - width + 1, string.length)}` : string;
```

Reply:
extract `\u2026` to const

Reply:
also this would probably be cleaner using slice